src/Controller/Http4xxController.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Drupal\localgov_core\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+
+/**
+ * Returns responses for LocalGov Core routes.
+ */
+class Http4xxController extends ControllerBase
+{
+
+  /**
+   * The default 403 content.
+   *
+   * @return array
+   *   A render array containing the message to display for 403 pages.
+   */
+  public function on403()
+  {
+    return [
+        'message' => [
+            '#type' => 'html_tag',
+            '#tag' => 'p',
+            '#value' => $this->t('You are not authorised to access this page.')
+        ],
+        'login_link' => [
+            '#type' => 'html_tag',
+            '#tag' => 'a',
+            '#attributes'=> [
+                'href'=> '/user/login'
+            ],
+            '#value' => $this->t('Login?')
```

Comment:
When we added a login link to our 403 pages, we found customers trying to login with their MyAccount details so I'd suggest not including this.

Reply:
That's true. 

However it's just what's in the hard-coded templates at the moment: https://github.com/localgovdrupal/localgov_base/blob/1.x/templates/layout/status-pages/page--403.html.twig

It's also not ideal because someone could change that url for user.login in route and then that path would no longer work.

So I'm for removing it but decided not to get too opinionated with my changes.